src/hooks/useNavbarRoutes.ts

```diff
@@ -9,6 +9,10 @@ const navbarRoutes: NavbarItem[] = [
         href: "/student",
         label: "Om Sektionen",
       },
+      {
+        label: "Mentorskapsprogram",
+        href: "https://www.argz.se/mentorskapsprogram/",
+      },
```

Comment:
```suggestion
      {
        label: "Mentorskapsprogram",
        href: "https://www.argz.se/mentorskapsprogram/",
        newPage: true,
      },
```
Externa länkar bör öppna i en ny sida för bästa användarupplevelse